front/src/Phaser/Login/LogincScene.ts

```diff
@@ -23,30 +29,55 @@ export class LogincScene extends Phaser.Scene {
     }
 
     preload() {
-        this.load.image(LoginTextures.playButton, "resources/objects/play_button.png");
+        //this.load.image(LoginTextures.playButton, "resources/objects/play_button.png");
+        this.load.image(LoginTextures.icon, "resources/logos/tcm_full.png");
+        // Note: arcade.png from the Phaser 3 examples at: https://github.com/photonstorm/phaser3-examples/tree/master/public/assets/fonts/bitmap
+        this.load.bitmapFont(LoginTextures.mainFont, 'resources/fonts/arcade.png', 'resources/fonts/arcade.xml');
     }
 
     create() {
-        this.textField = new TextField(this, 10, 10, 'Enter your email:');
-        this.emailInput = new TextInput(this, 10, 50);
+        this.textField = new TextField(this, this.game.renderer.width / 2, 50, 'Enter your name:');
+        this.textField.setOrigin(0.5).setCenterAlign()
+        this.nameInput = new TextInput(this, this.game.renderer.width / 2 - 64, 70, 4);
 
-        let x = this.game.renderer.width / 2;
-        let y = this.game.renderer.height / 2;
-        this.playButton = new ClickButton(this, x, y, LoginTextures.playButton, this.login.bind(this));
+        this.pressReturnField = new TextField(this, this.game.renderer.width / 2, 130, 'Press enter to start');
+        this.pressReturnField.setOrigin(0.5).setCenterAlign()
+
+        //let x = this.game.renderer.width / 2;
+        //let y = this.game.renderer.height / 2;
+        //this.playButton = new ClickButton(this, x, y, LoginTextures.playButton, this.login.bind(this));
+
+        this.logo = new Image(this, this.game.renderer.width - 30, this.game.renderer.height - 20, LoginTextures.icon);
+        this.add.existing(this.logo);
+
+        let infoText = "Commands: \n - Arrows or Z,Q,S,D to move\n - SHIFT to run";
+        this.infoTextField = new TextField(this, 10, this.game.renderer.height - 35, infoText);
+
+        this.input.keyboard.on('keyup-ENTER', () => {
+            let name = this.nameInput.getText();
+            if (name === '') {
+                return
+            }
+            gameManager.connect(name).then(() => {
+                this.scene.start(GameSceneName, { name });
+            });
+        });
 
-        let infoText = "Commandes de base: \n - Z,Q,S,D (ou les flèches de direction) pour bouger\n - SHIFT pour accélerer";
-        this.infoTextField = new TextField(this, 10, 300, infoText);
     }
 
     update(time: number, delta: number): void {
-
+        if (this.nameInput.getText() == '') {
+            this.pressReturnField.setVisible(false);
+        } else {
+            this.pressReturnField.setVisible(!!(Math.floor(time / 500) % 2));
+        }
```

Comment:
alternatively, we could listen on nameInput for an event when the text is changed instead of checking every tick.

Reply:
True. But I need to make it blink (this is why I'm also depending on "time")